studio/schemas/documents/fleetCardSection.js

```diff
@@ -0,0 +1,13 @@
+export default {
+    name: 'fleetCardSection',
+    title: 'Fleet Card Section',
+    type: 'document',
+    fields: [
+        {
+            name: 'fleet',
+            title: 'Fleet',
+            type: 'array',
+            of: [{ type: 'reference', to: { type: 'fleetCard' } }],
+        },
+    ],
+};
```

Comment:
I don't think you'll need this either. All of this data can be queried at the index page level.